src/_tests_/RockShow.test.js

```diff
@@ -0,0 +1,35 @@
+import { render } from "@testing-library/react";
+import { MemoryRouter, Routes, Route } from "react-router-dom"
+import RockShow from "../Pages/RockShow";
+import mockRocks from "../mockRocks";
+
+describe("<RockShow />", () => {
+    it("renders without crashing", () => { })
+})
+it("renders a name", () => { 
+    render(
+        <MemoryRouter initialEntries={["/rockshow/1"]}>
+            <Routes>
+                <Route path="/rockshow/:id" element={<RockShow rocks={mockRocks}/>}/>
+            </Routes>
+        </MemoryRouter>
+        )
+})
+    it("renders an age", () => { 
+    render(
+        <MemoryRouter initialEntries={["/rockshow/1"]}>
+            <Routes>
+                <Route path="/rockshow/:id" element={<RockShow rocks={mockRocks}/>}/>
+            </Routes>
+        </MemoryRouter>
+        )
+    })
+    it("renders an enjoys", () => { 
+    render(
+        <MemoryRouter initialEntries={["/rockshow/1"]}>
+            <Routes>
+                <Route path="/rockshow/:id" element={<RockShow rocks={mockRocks}/>}/>
+            </Routes>
+        </MemoryRouter>
+        )
+    })
```

Comment:
Since these test cases are looking at features, you need an assertion for each one. FYI, assertions are expect statements.

Reply:
Assertions added